### DISCUSSION TITLE
cellular support for usb

### DISCUSSION THREAD

**Discussion by heini66** [freifunk-gluon/gluon#2989] [Ideas]
thxs for adding the cellular support.
it would be great to get support to any device that has enough flash to supprt usb cellular modems.

**Comment by blocktrron** [comment]
This is a deliberate design decision, as Gluon abstracts the high-level configuration (APN / Username / PIN) to the settings required for the specific modem protocol. Thus we need to know the modem-model when executing `gluon-reconfigure`.
